scripts/classification/imagenet/verify_pretrained.py

```diff
@@ -60,23 +64,26 @@
 normalize = transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
 
 """
-Aligning with TF implemenation, the crop-input ratio is 0.875
-Set the crop 342 for input size 299 accordingly. 
+Especially for inceptionv3, the input size has to be 299
+Set the inputsize to 299 even user set a different value
 """
-if 'inceptionv3' in model_name:
-    transform_test = transforms.Compose([
-        transforms.Resize(342, keep_ratio=True),
-        transforms.CenterCrop(input_size),
-        transforms.ToTensor(),
-        normalize
-    ])
-else:
-    transform_test = transforms.Compose([
-        transforms.Resize(256, keep_ratio=True),
-        transforms.CenterCrop(input_size),
-        transforms.ToTensor(),
-        normalize
-    ])
+if ('inceptionv3' in model_name) and input_size!=299:
```

Comment:
I suggest to omit this hard coded setting. Using different image size for performance comparison among different models is a valid application. Just pass the value as is.

Reply:
Thanks for prompt response, then if user does not set 299 for inceptionv3, is it okay for the runtime error prompt?  I mean the valid input size for inceptinv3/v4 should be 299, this can be found in the paper introducing the inception v3.

Reply:
It is a matter of implementation, and can be fixed if we modify
1. this line with `contrib.AdaptiveAvgPooling2D` https://github.com/hetong007/gluon-cv/blob/master/gluoncv/model_zoo/inception.py#L147
2. and this line with `GlobalAvgPool2D`: https://github.com/hetong007/gluon-cv/blob/master/gluoncv/model_zoo/inception.py#L188

You can either take the changes or leave it for now. Let's make this script flexible.

Reply:
@hetong007 Appreciate for the information, I'll investigate and remove the hard code from the script, thanks...